packages/stream_chat_flutter_core/pubspec.yaml

```
@@ -1,7 +1,7 @@
 name: stream_chat_flutter_core
 homepage: https://github.com/GetStream/stream-chat-flutter
 description: Stream Chat official Flutter SDK Core. Build your own chat experience using Dart and Flutter.
-version: 9.5.0
+version: 9.5.0+1
```

```suggestion
version: 9.5.0
```

We should only bump the version in release PR's